Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabDomUtils.cpp

```diff
@@ -64,6 +64,12 @@ namespace AzToolsFramework
                     settings.m_keepDefaults = true;
                 }
 
+                if ((flags & StoreInstanceFlags::StoreLinkIds) != StoreInstanceFlags::None)
+                {
+                    LinkIdMetadata linkIdMetadata;
+                    settings.m_metadata.Add(&linkIdMetadata);
```

Comment:
Nitpick: This can be simplified to `settings.m_metadata.Create<LinkIdMetadata>();`